src/lib/resolver.js

```diff
@@ -130,8 +131,9 @@ function resolveSelector(ctx, lang, args, expr, index) {
     return selector;
   }
 
-  const argValue = index[0].args ?
-    resolveIdentifier(ctx, lang, args, index[0].args[0].name)[1] : undefined;
+  const argName = index[0].args ? index[0].args[0].name : index[1];
+  const argValue = argName ?
+    resolveIdentifier(ctx, lang, args, argName)[1] : undefined;
```

Comment:
OK, I see now: the forked resolver only has `createEntry` and `createAttribute`.  `format` comes from the regular resolver.